services/src/test-library/src/main/java/org/openkilda/testing/Constants.java

```diff
@@ -27,6 +29,7 @@
     public static final Integer RULES_INSTALLATION_TIME = 5;
     public static final Integer STATS_LOGGING_TIMEOUT = 70;
     public static final SwitchId NON_EXISTENT_SWITCH_ID = new SwitchId("de:ad:be:ef:de:ad:be:ef");
+    public static final String NON_EXISTENT_FLOW_ID = "non-existent " + UUID.randomUUID().toString();
```

Comment:
Please, let's have the name without whitespace.

"non-existent " -> "non-existent-"